website/docs/docs/cloud/manage-access/self-service-permissions.md

```diff
@@ -1,42 +1,81 @@
 
-### Prerequisites
+## Table of groups, licenses, and permissions
 
-You need to create an Account Admins group before removing any other groups.
+Key:
 
-1. Create an Account Admins group.  
-2. Assign at least one user to the Account Admins group. The assigned user can manage future group, SSO mapping, and user or group assignment.
+* (W)rite &mdash; Create new or modify existing. Includes `send`, `create`, `delete`, `allocate`, `modify`, and `read`.
+* (R)ead &mdash; Can view but can not create or change any fields.
+* No value &mdash; No access to the feature.
 
-### Remove the Owner and Member groups
+Permissions:
 
-Follow these steps for both Owner and Member groups:
+* Account-level permissions &mdash; Permissions related to management of the dbt Cloud account. For example, billing and account settings.
+* Project-level permissions &mdash; Permissions related to the projects in dbt Cloud. For example, Explorer and the IDE.
+
+The following tables outline the access that users have if they are assigned a Developer license and the Owner or Member group, Read-only license, or IT license.
+
+#### Account permissions for account roles
+
+| Account-level permission| Owner | Member | Read-only license|  IT license  |
+|:------------------------|:-----:|:------:|:----------------:|:------------:|
+| Account settings        |   W   |   W    |                  |       W      |
+| Billing                 |   W   |        |                  |       W      |
+| Invitations             |   W   |   W    |                  |       W      |
+| Licenses                |   W   |   R    |                  |       W      |
+| Users                   |   W   |   R    |                  |       W      |
+| Project (create)        |   W   |   W    |                  |       W      |
+| Connections             |   W   |   W    |                  |       W      |
+| Service tokens          |   W   |        |                  |       W      |
+| Webhooks                |   W   |   W    |                  |              |
+ 
+#### Project permissions for account roles
+
+|Project-level permission | Owner | Member  | Read-only | IT license |
+|:------------------------|:-----:|:-------:|:---------:|:----------:|
+| Adapters                |   W   |    W    |    R      |             |
+| Connections             |   W   |    W    |    R      |             |
+| Credentials             |   W   |    W    |    R      |             |
+| Custom env. variables   |   W   |    W    |    R      |             |
+| dbt adapters            |   W   |    W    |           |             |
+| Develop (IDE or dbt Cloud CLI)| W |  W    |           |             |
+| Environments            |   W   |    W    |    R      |             |
+| Jobs                    |   W   |    W    |    R      |             |
+| Metadata                |   R   |    R    |    R      |             |
```

Comment:
by metadata, do we mean explorer?

Reply:
I'm not sure. This was already in the table when I moved it. It's all read-only so I don't think it's Explorer as a whole, but I'll remove it for now and revisit when we update these tables